src/languages/ruby.md

```diff
@@ -4,8 +4,8 @@ Platform.sh supports deploying any Ruby application. Your application can use
 any Ruby application server such as Unicorn or Puma and deploying a Rails or
 a Sinatra app is very straight forward.
 
-### Supported versions
-#### Ruby MRI
+## Supported versions
```

Comment:
In any case, formatting is wrong so let's at least fix that. And since it's the only one we support and that's the one that everybody think of by default :), we should remove it.